pipreqs/pipreqs.py

```diff
@@ -473,20 +513,21 @@ def init(args):
         # the list of exported modules, installed locally
         # and the package name is not in the list of local module names
         # it add to difference
-        difference = [x for x in candidates if
-                      # aggregate all export lists into one
-                      # flatten the list
-                      # check if candidate is in exports
-                      x.lower() not in [y for x in local for y in x['exports']]
-                      and
-                      # check if candidate is package names
-                      x.lower() not in [x['name'] for x in local]]
-
-        imports = local + get_imports_info(difference,
-                                           proxy=proxy,
-                                           pypi_server=pypi_server)
+        difference = [
+            x
```

Comment:
formatters are weird

Reply:
What should we do in this case? Even if I solved this problem manually fixing this formater weirdness, next time someone runs the formatter it will just go back to this weird thing...

Reply:
@Fernando-crz first thing would be to understand why is it doing this haha are we sure this happens in the latest version of black? what are the exact options set? can anyone replicate this weird formatting?

Reply:
23.9.1^ output of black version with flag for 120 character lines

Reply:
I suggest that we can use darker instead (https://github.com/akaihola/darker/tree/master). It only runs the formatter over changes in the commits, meaning that, if I change this formatting back to it's origin state, it won't be changed in the next commits to the "next" branch.

By now, I will just revert the changes related to this specific line.